### ISSUE TITLE
CentOS 8 Support

### ISSUE THREAD

**Issue by micafer** [NVIDIA/ansible-role-nvidia-driver#36]

**Comment by l6l** [comment]
I used this playbook to get past the GPG error:
```
- name: install keys
  hosts: worker
  tasks:
  - rpm_key:
      state: present
      key: https://rpms.remirepo.net/RPM-GPG-KEY-remi2018
  - rpm_key:
      state: present
      key: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
  - name: Install epel repo.
    yum:
      name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
      state: present
  - name: Install remi repo.
    yum:
      name: "http://rpms.remirepo.net/enterprise/remi-release-8.rpm"
      state: present

- name: install CUDA driver on all GPU nodes
  hosts: worker
  roles:
    - nvidia.nvidia_driver
```

**Comment by l6l** [comment]
systemd must be updated to the latest version on the target GPU nodes, otherwise Ansible may fail to find the persistence daemon.

**Comment by dholt** [comment]
Should be fixed in #45
